front_end/src/components/Accounts/AccountDetails/AccountDetails.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../../styles/variables';
+
+.details-panel.account-details{
+  /*strong{
+    color: $color-gray;
+    font-weight: 600;
+  }
+
+  .account-details-top{
+    padding: 15px;
+    font-weight: 600;
+  }*/
```

Comment:
nothing here. Is this placeholder for something coming or should the file be removed?